RevenueCatUI/Data/TemplateViewConfiguration.swift

```diff
@@ -30,6 +30,7 @@ extension TemplateViewConfiguration {
 
         let content: RevenueCat.Package
         let localization: ProcessedLocalizedConfiguration
+        let discountRelativeToMostExpensive: Double?
```

Comment:
Is this actually `discountRelativeToMostExpensivePerMonth`? It's a long variable name but just wanted to make sure its clear 😛